fe/fe-core/src/main/java/com/starrocks/catalog/StructField.java

```diff
@@ -26,10 +26,13 @@
  * comments of struct fields. We set comment to null to avoid compatibility issues.
  */
 public class StructField {
+    // If name is null, that means this StructFiled is unnamed.
     @SerializedName(value = "name")
     private final String name;
     @SerializedName(value = "type")
     private final Type type;
+
+    // comment is not used now, it's always null.
```

Comment:
just remove it

Reply:
`comment` field has existed in thrift for a long time, we should keep it.